server/admin_management/api/app/schemas.py

```diff
@@ -339,4 +344,5 @@ class GCNotifyGrantDelegatedAdminEmailParam(BaseModel):
     last_name: Annotated[str, StringConstraints(max_length=20)]
     role_list_string: Annotated[str, StringConstraints(max_length=200)]
     application_team_contact_email: Optional[EmailStr] = None
+    with_client_number: Literal['yes', 'no']
```

Comment:
This is new GC Notify's variable for conditional display (it needs value as 'yes' or 'no').

Reply:
thanks Ian, later on once we support grant multiple roles might need to look at this again, good for now